io.github.KikoPlayProject.KikoPlay.yml

```diff
@@ -0,0 +1,342 @@
+app-id: io.github.KikoPlayProject.KikoPlay
+branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: KikoPlay
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  # aria2c download
+  - --filesystem=xdg-videos
+  - --filesystem=xdg-download
+  # sound
+  - --filesystem=xdg-run/pipewire-0:ro
+  - --socket=x11
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --persist=.config/kikoplay
+  - --env=LC_NUMERIC=C
+
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /lib/debug
+  - /man
+  - /share/man
+
+modules:
+  - name: KikoPlay
+    buildsystem: qmake
+    config-opts:
+      - LIBS+=-L/app/lib
+    sources:
+      - type: archive
+        url: https://github.com/KikoPlayProject/KikoPlay/archive/refs/tags/1.0.0.tar.gz
+        sha256: a8e20f9880fc3418e12f836c28684607e6486d1d2e504e7e829daa5c2344a47f
+      - type: patch
+        path: patch/0001-prebuild.patch
+      - type: patch
+        path: patch/0002-install.patch
```

Comment:
You shouldn't hardcode any of these, let it choose from prefix/path etc.

Reply:
I wonder if it will work if you can make them relative to PREFIX then run qmake as `PREFIX=/app qmake`

```
target.path = $$(PREFIX)
INSTALLS += target
DEFINES += INSTALL_PREFIX=$$(PREFIX)
```

Reply:
it's qmake. that kind of patch is fine downstream.